be/src/exec/hdfs_scanner.h

```diff
@@ -321,6 +321,8 @@ struct HdfsScannerContext {
 
     int64_t connector_max_split_size = 0;
 
+    OlapRuntimeScanRangePruner* rf_scan_range_pruner = nullptr;
```

Comment:
we definitely have to remove "Olap"

Reply:
Good suggestion, i will remove it next pr.